src/modules/auth/ui/views/sign-up-view.tsx

```
@@ -1,7 +1,248 @@
+                </p>
+              </div>
+              {!!error && (
+                <Alert className="bg-destructive/10 text-destructive border-none">
+                  <OctagonAlertIcon className="h-4 w-4" />
+                  <AlertTitle>{error}</AlertTitle>
+                </Alert>
+              )}
+              <div className="grid gap-3">
+                <form.Field
+                  name="name"
+                  children={(field) => (
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)